setup.py

```diff
@@ -31,7 +31,7 @@
     ),
     package_data={"pyhyp": ["*.so"]},
     install_requires=[
-        "numpy>=1.16,<1.24",
+        "numpy>=1.16,!=1.24,!=1.24.1,!=1.24.2",
```

Comment:
Do we need `!=1.24`? I would think `!=1.24.1,!=1.24.2` is enough?

Reply:
No, we need it since version 1.24.0 (and some RCs) would be matched otherwise.